api/database/migration_exif.go

```diff
@@ -68,13 +112,21 @@ func migrateExifFieldsExposure(db *gorm.DB) error {
 			return errors.Wrapf(err, "convert flash attribute empty values to NULL")
 		}
 
-		type exifModel struct {
-			ID       int `gorm:"primarykey"`
-			Exposure *string
-		}
 		var results []exifModel
 
-		return tx.Model(&exifModel{}).Table("media_exif").Where("exposure LIKE '%/%'").FindInBatches(&results, 100, func(tx *gorm.DB, batch int) error {
+		return calculateExposure(tx, results)
+	})
+
+	if err != nil {
+		return errors.Wrap(err, "migrating `media_exif.exposure` failed")
```

Comment:
Since we are changing this line, it's better to follow best practices of error.

```
return fmt.Errorf("migrating `media_exif.exposure` failed: %w", err)
```

Reply:
Thanks, I'll change it, but could you please explain the difference between these 2 expressions and when is it better to use which?
I'll then go through other PRs in advance, as we have such calls across all the project.

Reply:
[`fmt.Errorf()`](https://pkg.go.dev/fmt#Errorf) is an official verison of `errors.Warpf()`. They do the same thing, to attach a context info to an error. `fmt.Errorf()` works like `fmt.Printf()` and returns an error. It introduces a new placeholder `%w` to wrap an error, and put the string returned by `error.Error()`.

In the other hand, https://github.com/pkg/errors is already archived.

Reply:
Done in all the module, please review